core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -184,3 +184,100 @@ oppia.directive('trainingPanel', [
     };
   }
 ]);
+
+oppia.directive('trainingPanel', [function() {
```

Comment:
@bching `trainingPanel` directive is already there right? I guess you added a second copy here.

Reply:
@shubha1593 Sorry, it looks like I somehow did. Removed now.